x509-cert/src/macros.rs

```diff
@@ -1,5 +1,7 @@
 //! Macros used by this crate
 
+/// macro_rules implementation of former newtype proc macro. See `<https://github.com/RustCrypto/formats/pull/626>`.
```

Comment:
I think it might be helpful to describe what the macro actually does.

```suggestion
/// Implements the following traits for a newtype of a `der` decodable/encodable type:
///
/// - `From` conversions to/from the inner type
/// - `AsRef` and `AsMut`
/// - `DecodeValue` and `EncodeValue`
/// - `FixedTag` mapping to the inner value's `FixedTag::TAG`
///
/// The main case is simplifying newtypes which need an `AssociatedOid`
```

Reply:
Changed.